qiskit_ibm_runtime/api/clients/utils.py

```diff
@@ -61,8 +61,8 @@ def ws_proxy_params(client_params: ClientParameters, ws_url: str) -> Dict:
 
     if "auth" in conn_data:
         out["http_proxy_auth"] = (
-            client_params.proxies["username_ntlm"],
-            client_params.proxies["password_ntlm"],
+            client_params.proxies.username_ntlm,
+            client_params.proxies.password_ntlm,
```

Comment:
This function was created to retrofit the `proxies` parameters, which is in `requests` format, into websockets format. Since we now have a nice new `ProxyConfiguration` class, this should probably go into that class (e.g. as `to_ws_params()`) so we don't have proxy-related functions in multiple places.

Reply:
Good catch, didn't understand that indirection initially. Fixed via https://github.com/Qiskit/qiskit-ibm-runtime/commit/d94daf50f4e7b5a2b4698b14b1ba0cebfa72c205.